Xamarin.Forms.Core/BindingExpression.cs

```diff
@@ -148,7 +148,7 @@ void ApplyCore(object sourceObject, BindableObject target, BindableProperty prop
 
 				if (!TryConvert(ref value, property, property.ReturnType, true))
 				{
-					Log.Warning("Binding", "{0} can not be converted to type '{1}'", value, property.ReturnType);
+					Log.Warning("Binding", "'{0}' can not be converted to type '{1}'.", value, property.ReturnType);
```

Comment:
Please use string interpolation

Reply:
thanks for the suggestion, but we're in the process of moving the errors and warnings in a place where string.Format is better than string interpolation

Reply:
@VladislavAntonyuk @StephaneDelcroix personal opinion here.
If you leave the text at the source of the error, then when we as a user search for errors on the Google, we are taken to the source of the error. If you move the error elsewhere (say a Resource), then we have to do a second search to find the usage of the resource key.

just my $0.02

Reply:
Hmmm that's a good point. We have to move them because of localization requirements; however, it might be useful to leave the text in a comment here. It might get out of sync, though.

Reply:
@samhouts it will very likely get out of sync. Perhaps (and this would be a much larger task) Forms can start using unique error codes that are specified at the source of the error. That way we don't care as much about the text as we do the code.

Reply:
we're in the process of doing just that https://github.com/xamarin/Xamarin.Forms/pull/10662 for build errors

helplinks is what comes next, hopefully